### PR TITLE
feat(job): overview for jobs

### PR DESCRIPTION
**Pull request by bdebon**
# What does this PR do?

- Created a `property-card` component to display this kind of property card value 
CleanShot 2022-12-27 at 15 17 36@2x

- Created a `JobOverview` component to gather everything about job without overloading too much the `page-general` component
- Unit testing

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-624

CleanShot 2022-12-27 at 15 18 37@2x

---

### Global

- [ ] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [ ] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [ ] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [ ] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

File: libs/pages/application/src/lib/ui/job-overview/job-overview.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  const { application } = props
+  const { organizationId = '', environmentId = '', applicationId = '', projectId = '' } = useParams()
+  const navigate = useNavigate()
+  const path = APPLICATION_URL(organizationId, projectId, environmentId, applicationId) + APPLICATION_SETTINGS_URL
+
+  const eventsToString = (): string => {
+    if (!application) return ''
+    const events: string[] = []
+    if (application) {
+      if (application.schedule?.on_start) events.push('Start')
```
Comment (RemiBonnet) [inline libs/pages/application/src/lib/ui/job-overview/job-overview.tsx:27]: You can't remove the `if(application) {` and add this one `if (application?.schedule?.on_start) events.push('Start')`  ?

File: libs/pages/application/src/lib/ui/job-overview/job-overview.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            name="Scheduling"
+            value={cronstrue.toString(application.schedule?.cronjob?.scheduled_at?.toString() || '')}
+            isLoading={false}
+            onSettingsClick={() => navigate(`${path}${APPLICATION_SETTINGS_CONFIGURE_URL}`)}
+          />
+        )}
+        {isLifeCycleJob(application) && (
+          <PropertyCard
+            name="Environment Event"
+            value={`${eventsToString()}`}
```
Comment (RemiBonnet) [inline libs/pages/application/src/lib/ui/job-overview/job-overview.tsx:49]: I think you can do it direcly `value={eventsToString()}`

**Comment by evoxmusic** [comment]
Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://zddf7f1aa-z4565356c-gtw.za8ad0659.bool.sh)
:point_right: [storybook](https://zddf7f1aa-z6e147f3b-gtw.za8ad0659.bool.sh)

**Review by RemiBonnet — COMMENTED** [review]
Well done @bdebon, I don't have important note !

**Pull request merged**